src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -11,6 +11,18 @@ namespace System.Management.Automation.Internal
 {
     internal static class ModuleUtils
     {
+        // Default option for local file system enumeration:
+        //  - Ignore files/directories when access is denied;
+        //  - Search top directory only.
+        private readonly static EnumerationOptions s_defaultEnumOptions = new EnumerationOptions() { AttributesToSkip = 0 };
+
+        // Default option for UNC path enumeration. Same as above plus a large buffer size.
+        // For network shares, a large buffer may result in better performance as more results can be batched over the wire.
+        private readonly static EnumerationOptions s_uncPathEnumOptions = new EnumerationOptions() { AttributesToSkip = 0, BufferSize = 16384 };
```

Comment:
Is there a reason for the buffer size to be this specific number (2^14)?

Reply:
It's recommended in the comments of `BufferSize`. I add a comment about it.